dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/subcommands/impl/Parent.java

```diff
@@ -50,6 +50,11 @@ public void execute(final Namespace ns, DockerfileGitHubUtil dockerfileGitHubUti
         log.info("Updating store...");
         this.dockerfileGitHubUtil.getGitHubJsonStore(ns.get(Constants.STORE)).updateStore(img, tag);
 
+        if (ns.get(Constants.SKIP_PR_CREATION)) {
+            log.info(String.format("Since the flag %s is set to True, the PR creation steps will " +
```

Comment:
Not need to `String.format`

Do this instead: 
`log.info("Since the flag {} is set to True, the PR creation steps will be skipped.", Constants.SKIP_PR_CREATION)`

Also, notice you have a trailing space after the period as well.